webpack/assets/javascripts/react_app/components/common/Alert.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import helpers from '../../common/helpers';
+import CloseButton from '../common/CloseButton';
+import { ALERT_CSS } from '../../constants';
+
+class Alert extends React.Component {
+  constructor(props) {
+    super(props);
+    helpers.bindMethods(this, ['dismiss']);
+    this.state = { visible: true };
+  }
+
+  dismiss() {
+    this.setState({ visible: false });
+  }
+
+  render() {
+    const { type, dismissable, children } = this.props;
+
+    return (
+      this.state.visible ?
```

Comment:
No, that won't work. I need the expression to return null if `this.state.visible` is false. The expression you suggest would return false.
Alert is not used and has been removed.